app/view_objects/apply_invalid_data_view.rb

```diff
@@ -32,9 +32,11 @@ def summary_items_content
   attr_reader :apply_application, :invalid_fields
 
   def populate_invalid_fields
+    return {} if apply_application.invalid_data.blank?
+
     fields = []
 
-    apply_application.invalid_data.each do |section_key, field_and_values|
+    JSON(apply_application.invalid_data).each do |section_key, field_and_values|
```

Comment:
Should we consider `jsonb_accessor` gem?

Or at the very least centralise `JSON(apply_application.invalid_data)` in their models

Reply:
this is fixed, thanks!